public/locales/en/translation.json

```
@@ -22,7 +22,8 @@
     "configureDaemon": "Make sure you <1>configure your IPFS API</1> to allow cross-origin (CORS) requests:",
     "runDaemon": "Then, start an IPFS daemon in your terminal:",
     "footNote": "If no other nodes pin your files and you stop serving them, they will disappear from the network.",
-    "largeFilesWarning": "If you experience issues when downloading files larger than 1GB, please install <1>IPFS Desktop</1> or <3>IPFS Companion</3> for improved performance."
+    "largeFilesWarning": "If you experience issues when downloading files larger than 1GB, please install <1>IPFS Desktop</1> or <3>IPFS Companion</3> for improved performance.",
+    "openInGateway": "Open in the gateway"
```

To avoid inventing new sentences and re-use translation I copied one from IPLD Explorer:

```suggestion
    "openInGateway": "View on IPFS Gateway"
```